Directory.Build.props

```diff
@@ -4,7 +4,10 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Target Platforms" >
-    <NetCoreVersions>net7.0;net8.0</NetCoreVersions>
+    <LatestSupportedDotNetVersion>net8.0</LatestSupportedDotNetVersion>
+    <OldestSupportedDotNetVersion>net7.0</OldestSupportedDotNetVersion>
+    <NetCoreVersions>LatestSupportedDotNetVersion</NetCoreVersions>
```

Comment:
You're missing `$()` around the property

Reply:
```suggestion
    <NetCoreVersions>${LatestSupportedDotNetVersion)</NetCoreVersions>
```